components/tor/pref_names.h

```diff
@@ -6,15 +6,15 @@
 #ifndef BRAVE_COMPONENTS_TOR_PREF_NAMES_H_
 #define BRAVE_COMPONENTS_TOR_PREF_NAMES_H_
 
-namespace tor {
-namespace prefs {
+namespace tor::prefs {
```

Comment:
is this allowed in chromium now?

Reply:
https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++-features.md#Nested-namespaces-allowed